app/code/Magento/InventoryShipping/Ui/DataProvider/SourceSelectionDataProvider.php

```diff
@@ -143,27 +162,48 @@ public function getData()
                 'sku' => $sku,
                 'product' => $this->getProductName($orderItem),
                 'qtyToShip' => $qty,
-                'sources' => $this->getSources($websiteId, $sku, $qty),
+                'sources' => $this->getSources($stockId, $sku, $qty),
```

Comment:
In the `protected function getProductName(Item $item)` which id also protected by the way.

You mix representation logic (using HTML tags) with data preparation. Which is wrong from the separation of concept point of view.